kiln/devnets/goerlishadow-4.vars

```diff
@@ -0,0 +1,21 @@
+DEVNET_NAME=goerlishadow-4
+
+GETH_IMAGE=parithoshj/geth:master
+NETHERMIND_IMAGE=nethermindeth/nethermind:kiln_0.12
```

Comment:
need to check these images with Paritosh (Paritosh is building/maintaining the geth builds)/Marek

Reply:
Updated nethermind image. Confirmed with Pari to use `geth:master` at this time.